.github/workflows/tests.yml

```diff
@@ -7,7 +7,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [3.8, 3.7, 3.6]
-        plone-version: ["6.0", "5.2"]
```

Comment:
@ale-rt as outlined in my comment I'd prefer to keep things as they are in tests.yml. The rest of the PR is fine. :)

Reply:
I know what you outlined, but I do not understand it :)
jenkins tests are already doing everything is needed and testing Plone 6.0 here is just adding complications.
In addition the tested Plone 6 version is already old, see:
- https://github.com/plone/plone.restapi/blob/52b2bcfbffa4559216843c37a19d088fd0e722f8/plone-6.0.x.cfg#L4

I am still convinced this PR is doing the right thing!